Mathlib/Algebra/Lie/CartanMatrix.lean

```diff
@@ -66,23 +66,24 @@ exceptional semisimple Lie algebras.
 
 ## Main definitions
 
-  * `matrix.to_lie_algebra`
-  * `cartan_matrix.E₆`
-  * `cartan_matrix.E₇`
-  * `cartan_matrix.E₈`
-  * `cartan_matrix.F₄`
-  * `cartan_matrix.G₂`
-  * `lie_algebra.e₆`
-  * `lie_algebra.e₇`
-  * `lie_algebra.e₈`
-  * `lie_algebra.f₄`
-  * `lie_algebra.g₂`
+  * `Matrix.ToLieAlgebra`
+  * `CartanMatrix.E₆`
+  * `CartanMatrix.E₇`
+  * `CartanMatrix.E₈`
+  * `CartanMatrix.F₄`
+  * `CartanMatrix.G₂`
+  * `LieAlgebra.E₆`
+  * `LieAlgebra.E₇`
+  * `LieAlgebra.E₈`
+  * `LieAlgebra.F₄`
+  * `LieAlgebra.G₂`
```

Comment:
I'm very strongly of the opinion that these should remain lowercase. It is standard practice in mathematics to use:

- uppercase for the cartan matrix
- lowercase for the Lie algebra
- uppercase for the Lie group (but they aren't in mathlib yet)

Reply:
Oops, I didn't realize I'd done this. Fixed.